ChangeLog.md

```diff
@@ -11,7 +11,9 @@
   - `GetBakerEarliestWinTime`
   - `GetWinningBakersEpoch`
   - `GetFirstBlockEpoch`
-- Add support for `CommissionRates` in `CurrentPaydayBakerPoolStatus` (Only available for node versions > 6.0).
+  - Add support for `CommissionRates` in `CurrentPaydayBakerPoolStatus` (Only    available for node versions > 6.0).
+  - By default show all options for importing an account.
```

Comment:
There is no "default" anymore. it is not possible to disable this behaviour.

Reply:
Good point.